scripts/remove-low-rated-notes

```diff
@@ -0,0 +1,38 @@
+#!/usr/local/bin/php -q
+<?php
+
+use App\DB;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+define('RATING_THRESHOLD', "-5");
+define('AGE_THRESHOLD', '1 year');
+
+$pdo = DB::connect();
+
+$query = "SELECT COUNT(*) FROM note";
+$total = $pdo->single($query);
+
+/** --[ the lowest rated notes ]------------------- **/
+
+$date = date('Y-m-d', strtotime(AGE_THRESHOLD . ' ago'));
+$query = "SELECT sect, note.ts, note_id, SUM(if (vote = 0, -1, 1)) AS weight FROM note, votes WHERE note.ts < '{$date}' AND note.id = votes.note_id GROUP by note_id HAVING weight < " . RATING_THRESHOLD . " ORDER BY weight";
```

Comment:
$date should be parameterized. I know it's safe but it's cleaner when it's separate.